taf/settings.py

```diff
@@ -16,6 +16,10 @@
 
 default_branch = "main"
 
+# Strict mode enabled/disabled. If error_if_warning is enabled, any warnings
+# should raise TAF errors
+error_if_warning = False
```

Comment:
I actually like the name `strict` a lot more. `error_if_warning` sounds contradictory - if it's a warning, it's a warning, if it's an error, it's an error.

Reply:
That makes sense, renamed `error_if_warning` to `strict`